worker/uniter/charm/charm.go

```diff
@@ -39,7 +38,10 @@ type Bundle interface {
 type BundleInfo interface {
 
 	// URL returns the charm URL identifying the bundle.
-	URL() *charm.URL
+	//URL() *charm.URL
```

Comment:
Drop comment if this is actually being removed.

Reply:
This comment is relevant to pr https://github.com/juju/juju/pull/14174, will Resolve there.